tests/test_squiral.py

```diff
@@ -5,7 +5,7 @@
 
 
 def test_version():
-    assert __version__ == '0.1.1'
```

Comment:
Well, it's already been 0.1.2
https://github.com/sadikkuzu/squiral/blob/main/tests/test_squiral.py#L8